Tests/DatadogTests/Datadog/CrashReporting/CrashReporterTests.swift

```diff
@@ -58,7 +58,7 @@ class CrashReporterTests: XCTestCase {
         let crashReporter = CrashReporter(
             crashReportingPlugin: plugin,
             crashContextProvider: CrashContextProviderMock(),
-            loggingOrRUMIntegration: integration,
+            integration: integration,
             messageReceiver: NOPFeatureMessageReceiver()
         )
```

Comment:
☝️ Similar to above. IMO it would make sense for us to rename the test:
```diff
- testGivenNoPendingCrashReport_whenLoggingOrRUMIntegrationIsEnabled_itDoesNotSendTheCrashReport
+ testWhenPendingCrashReportIsNotFound_itDoesNothing
```